samples/bluetooth/direct_test_mode/src/fem/nrf21540.c

```diff
@@ -71,27 +71,18 @@
 #define RX_EN_PIN DT_GPIO_PIN(NRF21540_NODE, rx_en_gpios)
 #define RX_EN_FLAGS DT_GPIO_FLAGS(NRF21540_NODE, rx_en_gpios)
 
-#define PDN_PORT DT_GPIO_LABEL(NRF21540_NODE, pdn_gpios)
-#define PDN_PIN DT_GPIO_PIN(NRF21540_NODE, pdn_gpios)
-#define PDN_FLAGS DT_GPIO_FLAGS(NRF21540_NODE, pdn_gpios)
-
-#define ANT_SEL_PORT DT_GPIO_LABEL(NRF21540_NODE, ant_sel_gpios)
-#define ANT_SEL_PIN DT_GPIO_PIN(NRF21540_NODE, ant_sel_gpios)
-#define ANT_SEL_FLAGS DT_GPIO_FLAGS(NRF21540_NODE, ant_sel_gpios)
-
 #define PDN_SETTLE_TIME DT_PROP(NRF21540_NODE, pdn_settle_time_us)
 #define TX_EN_SETTLE_TIME DT_PROP(NRF21540_NODE, tx_en_settle_time_us)
 #define RX_EN_SETTLE_TIME DT_PROP(NRF21540_NODE, rx_en_settle_time_us)
 
-#if defined(NRF5340_XXAA_NETWORK)
+#if defined(NRF5340_XXAA_NETWORK) && DT_NODE_HAS_PROP(NRF21540_NODE, spi_if)
```

Comment:
I wonder about using `NRF21540_FEM_SPI_SUPPORTED` instead of `DT_NODE_HAS_PROP(NRF21540_NODE, spi_if)`. Just a suggestion:)

Reply:
Good point! Done